src/runtime/crdt/crdt.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+export type VersionMap = Map<string, number>;
+
+// A CRDT model is parameterized by:
```

Comment:
Style question: do we use //-style comments or /** */- style comments for documentation? I would normally use /** */ to document classes, functions, etc. and use // for implementation comments.